server/webhook.go

```diff
@@ -64,6 +64,10 @@ func (wh webhook) PostToChannel(p *Plugin, instanceID types.ID, channelId, fromU
 		text = p.replaceJiraAccountIds(instanceID, wh.text)
 	}
 
+	if p.getConfig().DisplaySubscriptionNameInNotifications && wh.text != "" && subscriptionName != "" {
+		text = fmt.Sprintf("%s\nSubscription Name: %s", text, subscriptionName)
```

Comment:
@aaronrothschild Waiting for what it looks like in a screenshot, but it looks like the post currently has the subscription name at the bottom like:

```
Michael commented on ticket A

Here's my comment

Subscription Name: Michael's Subscription
```

I'm thinking the subscription name should be at the top of the post, what do you think?

Reply:
> I'm thinking the subscription name should be at the top of the post, what do you think? 

Hmm, I'm 0/5 because I think the activity is important, and the subscription is useful for torubleshooting or understanding why a notification pops up in a channel. So, I could see it belonging at the end of the message too.  @abhijit-singh

Reply:
Okay then I suppose it should be at the end of the post